pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -669,15 +661,13 @@ void Panels::MemoryCardListPanel_Simple::UiCreateNewCard(McdSlotItem& card)
 	if (result != wxID_CANCEL)
 	{
 		card.IsEnabled = true;
-		card.Filename = dialog.result_createdMcdFilename;
+		card.Filename  = Path::FromWxString(dialog.result_createdMcdFilename);
 		card.IsPresent = true;
-		if (card.Slot >= 0)
-		{
-			Console.WriteLn(L"Setting new memory card to slot %u: '%s'", card.Slot, WX_STR(card.Filename.GetFullName()));
-		}
-		else
-		{
-			Console.WriteLn(L"Created a new unassigned memory card file: '%s'", WX_STR(card.Filename.GetFullName()));
+		wxString toPrint = Path::ToWxString(card.Filename);
+		if (card.Slot >= 0) {
+			Console.WriteLn(L"Setting new memory card to slot %u: '%s'", card.Slot, WX_STR(toPrint));
+		} else {
+			Console.WriteLn(L"Created a new unassigned memory card file: '%s'", WX_STR(toPrint));
```

Comment:
formatting

Side note: You may not be able to use `WX_STR` but you can probably use `.c_str` or something to get a `char*` from a `fs::path`